packages/pyright-internal/src/tests/fourslash/completions.override.explicitlyDisabled.fourslash.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference path="typings/fourslash.d.ts" />
+
+// @filename: pyrightconfig.json
+//// {
+////   "reportImplicitOverride": "none"
+//// }
+
+// @filename: test.py
+//// class B:
+////     def method_1(self, a: str, *args, **kwargs):
+////         pass
+////
+////     @property
+////     def method_2(self) -> int:
+////         pass
+////
+//// class C(B):
+////     def [|method/*marker*/|]
+
+// @ts-ignore
```

Comment:
```suggestion
// @ts-expect-error
```

I know all the other files use `@ts-ignore` but I prefer to use `@ts-expect-error` instead because it's a bit safer. See #86